test/topology_experimental_raft/test_tablets.py

```diff
@@ -1017,3 +1017,41 @@ async def test_storage_service_api_uneven_ownership_keyspace_and_table_params_us
         assert actual_ownership == pytest.approx(expected_ownerships[i], abs=delta)
 
         already_verified.add(actual_ip)
+
+@pytest.mark.asyncio
+async def test_tablet_storage_freeing(manager: ManagerClient):
```

Comment:
There's `test_tablet_cleanup_failure` in the very same file that also checks that no sstables are left. Do these two check different things? What is it?

Also there's `test_tablet_cleanup` that seem to _just_ check that data is not resurrected. Does it make sense to extend this test to also check sstables? Or is there some difference here that deserves a whole new test?

Reply:
This was written by Michal to verify that we're not leaving fds opened, after cleanup, which will prevent disk space from being freed. While IIRC tablet_cleanup and failure variant are more concerned about data consistency (e.g. resurrection) if a sstable still exists in the file system.

Reply:
This sounds like those three tests do the same -- check that $something didn't leak after cleanup :) If that's the case, it's worth having one, probably parametrized, test

Reply:
Right, we can merge those tests, they probably aren't carrying their own weight.